packages/components/buttons/primary-button/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@commercetools-uikit/primary-button",
-  "description": "Primary buttons are used for a primary action on a page.",
+  "description": "Primary buttons are used for a primary action on a page. You must also pass a label for accessibility reasons.",
```

Comment:
I remember nicola mentioning that detailed description can be moved to `description.md` file